### PR TITLE
Log the active hardware configuration

### LINKED ISSUE #2081: Log the active hardware configuration
Number of CPUs, amount of RAM, etc. should be visible in logs for debugging purposes.

### PR DESCRIPTION
**Pull request by etam**
fixes #2081

### CODE REVIEW

File: golemapp.py [hunk 1]
```diff
@@ -14,6 +19,8 @@
 from golem.core.variables import PROTOCOL_CONST
 from golem.node import Node
 
+log = logging.getLogger('golem.version')
```
Comment (maaktweluit) [inline golemapp.py]: Now this logger no longer only applies to the version maybe we can change it to `__name__`?

Comment (maaktweluit) [inline golemapp.py]: `logger` is more popular in golem

**Review by maaktweluit — APPROVED** [review]
LGTM!

2 small comments, not blocking so approve!

**Comment by mfranciszkiewicz** [comment]
Could we also log the active preset configuration (e.g. on change)? IMO that would allow us to have some insight to why some jobs are failing.

**Comment by etam** [comment]
> Could we also log the active preset configuration (e.g. on change)?

done

**Review by mfranciszkiewicz — APPROVED** [review]

**Comment by etam** [comment]
do we want this also in b0.12.0?

**Pull request merged**